pkgs/development/interpreters/lambda-lisp/default.nix

```diff
@@ -0,0 +1,72 @@
+# Lambda Lisp has several backends, here we are using
+# the blc one. Ideally, this should be made into several
+# packages such as lambda-lisp-blc, lambda-lisp-lazyk,
+# lambda-lisp-clamb, etc.
+
+{ lib
+, gccStdenv
+, fetchFromGitHub
+, fetchurl
+, runtimeShell
+}:
+
+let
+  stdenv = gccStdenv
+  s = import ./sources.nix { inherit fetchurl fetchFromGitHub; };
+in
+stdenv.mkDerivation rec {
+  pname = "lambda-lisp-blc";
+  version = s.lambdaLispVersion;
+  src = s.src;
+  flatSrc = s.flatSrc;
+  blcSrc = s.blcSrc;
+
+  installPhase = ''
+    runHook preInstall
+    make -j $NIX_BUILD_CORES $makeFlags
```

Comment:
Are you sure this needs to be in `installPhase` and manually done instead of letting `buildPhase` do it's normal thing

Reply:
I'm honestly not, should I just try to remove the `make` line here?

Reply:
yeah

Reply:
seems to work